src/routes/safe/components/Apps/components/ConfirmTransactionModal.tsx

```diff
@@ -133,27 +140,25 @@ const ConfirmTransactionModal = ({
     <>
       <AddressInfo ethBalance={ethBalance} safeAddress={safeAddress} safeName={safeName} />
       <DividerLine withArrow />
-      {txs.map((tx, index) => {
-        return (
-          <Wrapper key={index}>
-            <Collapse description={<AddressInfo safeAddress={tx.to} />} title={`Transaction ${index + 1}`}>
-              <CollapseContent>
-                <div className="section">
-                  <Heading tag="h3">Value</Heading>
-                  <div className="value-section">
-                    <Img alt="Ether" height={40} src={getEthAsToken('0').logoUri} />
-                    <Bold>{humanReadableValue(tx.value, 18)} ETH</Bold>
-                  </div>
-                </div>
-                <div className="section">
-                  <Heading tag="h3">Data (hex encoded)*</Heading>
-                  <StyledTextBox>{tx.data}</StyledTextBox>
+      {txs.map((tx, index) => (
+        <Wrapper key={index}>
+          <Collapse description={<AddressInfo safeAddress={tx.to} />} title={`Transaction ${index + 1}`}>
+            <CollapseContent>
+              <div className="section">
+                <Heading tag="h3">Value</Heading>
+                <div className="value-section">
+                  <TokenLogo height={40} name={ethToken.name} logoUri={ethToken.logoUri as string} />
```

Comment:
why is it necessary to cast the `logoUri` to a `string` ... according to the type dev it should be already one, right?